beacon-chain/execution/engine_client.go

```diff
@@ -828,8 +790,8 @@ func fullPayloadFromPayloadBody(
 		if err != nil {
 			return nil, errors.Wrap(err, "unable to extract BlobGasUsed attribute from execution payload header")
 		}
-		return blocks.WrappedExecutionPayloadDeneb(
-			&pb.ExecutionPayloadDeneb{
+		return blocks.WrappedExecutionPayloadElectra(
+			&pb.ExecutionPayloadElectra{
```

Comment:
We are missing `DepositReceipts` and `WithdrawalRequests` here. I think we will need an `ExecutionPayloadBodyV2` and a new version of `engine_getPayloadBodiesByRoot` to accompany that

Reply:
This is going to be a bigger change and still a little ambiguous with upstream specs. Can we handle this in a follow-up PR?